packages/preview/modern-cv/0.1.0/lib.typ

```diff
@@ -0,0 +1,546 @@
+/// - title (string): The title of the resume entry
+/// - location (string): The location of the resume entry
+/// - date (string): The date of the resume entry, this can be a range (e.g. "Jan 2020 - Dec 2020")
+/// - description (content): The body of the resume entry
+#let resume_entry(
+  title: none, 
+  location: "", 
+  date: "",
+  description: ""
+) = {
+  pad[
+    #justified_header(title, location)
+    #secondary_justified_header(description, date)
+  ]
+}
+
+/// Show cumulative GPA.
+/// *Example:*
+/// #example(`resume.resume_gpa("3.5", "4.0")`)
+#let resume_gpa(numerator, denominator) = {
+  set text(size: 12pt, style: "italic", weight: "light")
+  text[Cumulative GPA: #box[#strong[#numerator] / #denominator]]
+}
+
+/// Show a certification in the resume.
+/// *Example:*
+/// #example(`resume.resume_certification("AWS Certified Solutions Architect - Associate", "Jan 2020")`)
+/// - certification (content): The certification
+/// - date (content): The date the certification was achieved
+#let resume_certification(certification, date) = {
+  justified_header(certification, date)
+}
+
+/// Show a list of skills in the resume under a given category.
+/// - category (string): The category of the skills
+/// - items (list): The list of skills. This can be a list of strings but you can also emphasize certain skills by using the `strong` function.
+#let resume_skill_item(category, items) = {
+  set block(below: 0.65em)
+  set pad(top: 2pt)
+  
+  pad[
+    #grid(
+      columns: (20fr, 80fr),
+      gutter: 10pt,
+      align(right)[
+        #set text(hyphenate: false)
+        == #category
+      ],
+      align(left)[
+        #set text(size: 11pt, style: "normal", weight: "light")
+        #items.join(", ")
+      ],
+    )
+  ]
+}
+
+/// ---- End of Resume Template ----
+
+/// ---- Coverletter ----
+
+/// Cover letter template that is inspired by the Awesome CV Latex template by posquit0. This template can loosely be considered a port of the original Latex template. 
+/// This coverletter template is designed to be used with the resume template.
+/// - author (content): Structure that takes in all the author's information
+/// - profile_picture (image): The profile picture of the author. This will be cropped to a circle and should be square in nature.
+/// - date (date): The date the cover letter was created
+/// - accent_color (color): The accent color of the cover letter
+/// - body (content): The body of the cover letter
+#let coverletter(
+  author: (:), 
+  profile_picture: image,
+  date: datetime.today().display("[month repr:long] [day], [year]"),
+  accent_color: default_accent_color,
+  body
+) = {
+  set document(
+    author: author.firstname + " " + author.lastname, 
+    title: "resume",
+  )
+  
+  set text(
+    font: ("Source Sans Pro"),
+    lang: "en",
+    size: 11pt,
+    fill: color_darkgray,
+    fallback: false
```

Comment:
Why did you choose to disable font fallback? It helps users that need wide charsets for their CV, e.g. in Asia.